CHANGELOG.md

```diff
@@ -51,6 +51,7 @@
 
 ### New features
 
+* Add new `Lint/NoReturnInBeginEndBlocks` cop. ([@jcfausto][])
```

Comment:
This line is in the wrong place now due to rebasing. Move it to the **master (unreleased)** heading, and also add a **New features** section there.

Reply:
I see. I'll update that. 👍